### ISSUE TITLE
La mise à jour quotidienne ne se lance pas 

### ISSUE THREAD

**Issue by airvb** [vingerha/gazpar_2_mqtt#12]
**Which type of installation/run**
- docker-compose
- addon

**Your configuration**
GRDF_USERNAME= xxxx
GRDF_PASSWORD= yyyy
MQTT_HOST=192.168.1.200
MQTT_PORT=1883
MQTT_TOPIC=HA-gazpar
MQTT_CLIENTID=gaz
HASS_DISCOVERY=True
HASS_PREFIX=homeassistant
HASS_DEVICE_NAME=gazpar
SCHEDULE_TIME=06:30
DEBUG_MODE=True
INFLUXDB_ENABLE=True
INFLUXDB_HOST=192.168.1.200
INFLUXDB_ORG=oNe
INFLUXDB_BUCKET=gazpar
INFLUXDB_TOKEN=9xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx=
PRICE_KWH_DEFAULT=0.0961
PRICE_FIX_DEFAULT=0.77
PRICE_PATH=/data

**Describe the bug**
Malgré le paramètre  "SCHEDULE_TIME=06:30" la mise à jour n'est pas lancée
rien ds les logbook ...
....

**Additional**

**Comment by vingerha** [comment]
Effectivement, chez mon non plus...mais je l'utilise pas....curiuex car j'en suis sûre que je l'avais testé

**Comment by airvb** [comment]
Merci pour ta réponse,
Du coup, existe il une alternative pour automatiser la récupération des données ?

**Comment by vingerha** [comment]
not stale

**Comment by Gandulf78** [comment]
It is working fine in the fork here: https://github.com/Gandulf78/gazpar_2_mqtt

**Comment by vingerha** [comment]
Est-ce que tu peut faire un pull request?

**Comment by Gandulf78** [comment]
Je te fais cela.

**Comment by Gandulf78** [comment]
C'est fait. Premier PR de ma vie, j'avoue.

**Comment by vingerha** [comment]
Pas encore reçu :)

**Comment by vingerha** [comment]
En plus, j'attend que t'as fait des améliorations pour le container et le add-on et a fait des tests pour les deux... 
Je sais que c'est du boulot et LA raison pour que je ne fait pas trops des maj.

**Comment by Gandulf78** [comment]
> Pas encore reçu :)

Cette fois-ci ça devrait être bon.

**Comment by vingerha** [comment]
@airvb Si on change le docker-compose après la première installation (création du container), ça ne prend pas en compte les nouveaux paramètres. Donc il faut just enleve le container actuel et relancer le docker-compose. Je l'a re-testé deux foix. Peut-être dans le futur je vais changes ça mais l'effort de dev vs. le plus-value n'est pas top (pour moi)
